### PR TITLE
Changes for image export deduplication with tests #2757

### PR DESCRIPTION
**Pull request by joachimmetz**
Changes for image export deduplication with tests #2757

### CODE REVIEW

File: plaso/cli/image_export_tool.py [hunk 1]
```diff
@@ -678,17 +678,17 @@ def ParseArguments(self, arguments):
     argument_parser.add_argument(
         '--include_duplicates', dest='include_duplicates',
         action='store_true', default=False, help=(
-            'If extraction from VSS is enabled, by default a digest hash '
-            'is calculated for each file. These hashes are compared to the '
-            'previously exported files and duplicates are skipped. Use '
-            'this option to include duplicate files in the export.'))
+            'By default a digest hash (SHA-256)i is calculated for each file '
```
Comment (rgayon) [inline plaso/cli/image_export_tool.py:681]: 256)i => 256)

File: tests/end-to-end.py [hunk 2]
```diff
@@ -1159,13 +1150,61 @@ def __init__(
     self._image_export_path = None
     self._InitializeImageExportPath()
 
+  def _CompareHashesFile(self, test_definition, temp_directory):
```
Comment (rgayon) [inline tests/end-to-end.py:1153]: This was already added with #3102

Comment (rgayon) [reply tests/end-to-end.py:1153]: For some reason github didn't show the branch needed to be synced until after my comment

File: plaso/cli/image_export_tool.py (hunk 1 above)
Comment (joachimmetz) [reply plaso/cli/image_export_tool.py:681]: done

File: tests/end-to-end.py (hunk 2 above)
Comment (joachimmetz) [reply tests/end-to-end.py:1153]: Noworries, thx for flagging

**Comment by joachimmetz** [comment]
@rgayon comments addressed and PR updated to address merge conflict and open to dos

File: plaso/cli/image_export_tool.py [hunk 3]
```diff
@@ -760,9 +760,8 @@ def ParseOptions(self, options):
 
     self._ParseFilterOptions(options)
 
-    if (getattr(options, 'no_vss', False) or
```
Comment (rgayon) [inline plaso/cli/image_export_tool.py:763]: Do you confirm we want to drop the check against no_vss?

Comment (joachimmetz) [reply plaso/cli/image_export_tool.py:763]: Yes, for context see: https://github.com/log2timeline/plaso/issues/2757

**Pull request merged**